### PR TITLE
Fixing Gofmt errors

### PR DESCRIPTION
**Pull request by stack72**
These were continually showing as changes in my master branch

### CODE REVIEW

File: builtin/providers/google/resource_compute_instance.go [hunk 1]
```diff
@@ -4,11 +4,12 @@ import (
 	"fmt"
 	"log"
 
+	"strings"
```
Comment (jen20) [inline builtin/providers/google/resource_compute_instance.go]: This is still incorrect - if you take out line 6 it will be ok though.

**Comment by jen20** [comment]
Looks good with the exception of the comment inline. Thanks @stack72!

**Comment by phinze** [comment]
:neutral_face: sorry about that, and thanks for the fix!

I'd love to figure out a way to get Travis to fail the build if `gofmt

**Comment by phinze** [comment]
whoops - stray click when i realized @jen20 beat me to the punch! i'll step away slowly :blush:

**Comment by jen20** [comment]
Thanks @stack72!

**Comment by apparentlymart** [comment]
I guess I never did go back and see if I could figure out how to resolve #3462. But I agree with @phinze that it would be great to find a way to do it.

**Pull request merged**